ccd-definition/CaseField/CaseField-SDO-nonprod.json

```diff
@@ -0,0 +1,10 @@
+[

```

Comment:
Kindly review these changes, probably we dont need these chagnes

Reply:
Yes, not need to be changed for add this option. Many thanks Azam